From e2cd54e5b2b0aa0348cd505bf196c8c22d4bac76 Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Tue, 29 Aug 2017 15:13:21 +0300
Subject: [PATCH] Add missing dependancies

---
 tools/gn/bootstrap/bootstrap.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tools/gn/bootstrap/bootstrap.py b/tools/gn/bootstrap/bootstrap.py
index 004d81cef5a2..8996518a0a4c 100755
--- a/tools/gn/bootstrap/bootstrap.py
+++ b/tools/gn/bootstrap/bootstrap.py
@@ -457,6 +457,7 @@ def write_gn_ninja(path, root_gen_dir, options):
       'base/message_loop/message_pump_default.cc',
       'base/metrics/bucket_ranges.cc',
       'base/metrics/field_trial.cc',
+      'base/metrics/field_trial_params.cc',
       'base/metrics/field_trial_param_associator.cc',
       'base/metrics/histogram.cc',
       'base/metrics/histogram_base.cc',
@@ -507,10 +508,13 @@ def write_gn_ninja(path, root_gen_dir, options):
       'base/task_scheduler/scheduler_lock_impl.cc',
       'base/task_scheduler/scheduler_single_thread_task_runner_manager.cc',
       'base/task_scheduler/scheduler_worker.cc',
+      'base/task_scheduler/scheduler_worker_pool.cc',
       'base/task_scheduler/scheduler_worker_pool_impl.cc',
       'base/task_scheduler/scheduler_worker_pool_params.cc',
       'base/task_scheduler/scheduler_worker_stack.cc',
       'base/task_scheduler/scoped_set_task_priority_for_current_thread.cc',
+      'base/threading/scoped_blocking_call.cc',
+      'base/threading/scoped_blocking_call.h',
       'base/task_scheduler/sequence.cc',
       'base/task_scheduler/sequence_sort_key.cc',
       'base/task_scheduler/task.cc',
-- 
2.11.0

