From f1b22b18afebcd94638d945457c6e7bcaa8ac5ca Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Tue, 19 Sep 2017 11:39:33 +0300
Subject: [PATCH] Remove deprecated dependancies

---
 tools/gn/bootstrap/bootstrap.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/tools/gn/bootstrap/bootstrap.py b/tools/gn/bootstrap/bootstrap.py
index 9ff35791530f..ca27a50ceb52 100755
--- a/tools/gn/bootstrap/bootstrap.py
+++ b/tools/gn/bootstrap/bootstrap.py
@@ -481,8 +481,6 @@ def write_gn_ninja(path, root_gen_dir, options):
       'base/process/process_handle.cc',
       'base/process/process_iterator.cc',
       'base/process/process_metrics.cc',
-      'base/profiler/scoped_profile.cc',
-      'base/profiler/scoped_tracker.cc',
       'base/rand_util.cc',
       'base/run_loop.cc',
       'base/sequence_token.cc',
@@ -579,7 +577,6 @@ def write_gn_ninja(path, root_gen_dir, options):
       'base/trace_event/trace_log_constants.cc',
       'base/trace_event/tracing_agent.cc',
       'base/tracked_objects.cc',
-      'base/tracking_info.cc',
       'base/unguessable_token.cc',
       'base/value_iterators.cc',
       'base/values.cc',
-- 
2.11.0

