From 86472f35f70383e392735ae36a08436fcc3ec419 Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Fri, 11 Aug 2017 10:41:20 +0300
Subject: [PATCH] Remove deprecated includes

---
 tools/gn/bootstrap/bootstrap.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tools/gn/bootstrap/bootstrap.py b/tools/gn/bootstrap/bootstrap.py
index ff2ae57c46b0..55500addb2a2 100755
--- a/tools/gn/bootstrap/bootstrap.py
+++ b/tools/gn/bootstrap/bootstrap.py
@@ -550,8 +550,6 @@ def write_gn_ninja(path, root_gen_dir, options):
       'base/trace_event/memory_peak_detector.cc',
       'base/trace_event/memory_usage_estimator.cc',
       'base/trace_event/process_memory_dump.cc',
-      'base/trace_event/process_memory_maps.cc',
-      'base/trace_event/process_memory_totals.cc',
       'base/trace_event/sharded_allocation_register.cc',
       'base/trace_event/trace_buffer.cc',
       'base/trace_event/trace_config.cc',
-- 
2.11.0

