From ecdb8ebf65cc00ff7988e16c53bc262951e7e2f0 Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Fri, 22 Dec 2017 09:39:01 +0200
Subject: [PATCH] Use v8-qemu-wrapper.sh on v8 context snapshot creation.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
 tools/v8_context_snapshot/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/v8_context_snapshot/BUILD.gn b/tools/v8_context_snapshot/BUILD.gn
index 47de741cab8e..34568364f875 100644
--- a/tools/v8_context_snapshot/BUILD.gn
+++ b/tools/v8_context_snapshot/BUILD.gn
@@ -50,6 +50,7 @@ if (use_v8_context_snapshot) {
     output_path = rebase_path(output_file, root_build_dir)
 
     args = [
+      "./v8-qemu-wrapper.sh",
       "./" + rebase_path(
               get_label_info(
                       ":v8_context_snapshot_generator($v8_snapshot_toolchain)",
-- 
2.11.0

